std_msgs/Header header
sensor_msgs/CompressedImage left_image
sensor_msgs/CompressedImage right_image
sensor_msgs/PointCloud2 points